### PR TITLE
[MISC, DOC] Style changes and some documentation

### PR DESCRIPTION
**Pull request by Irallia**
I decided to write a small text for our GitHub starting page (README).
Also when I went throug the code I found some small things
and we spoke about a renaming of the sa_method into split_read_method, what I have done here aswell.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,28 @@
...
+David Heller in the Vingron lab of the MPI-MG and Tim White in the Kehr lab at BIH have both developed an SV caller for
+long read sequencing data. Instead of competing with each other, we want to join forces and combine the two tools, SVIM
+and SVIRL, into one better and more versatile tool.
+On the other hand, there were some tool developments in the Reinert Lab (FU Berlin): Vaquita, a short read SV caller,
+SViper, a refinement tool and Vaquita-LR a further development of Vaquita for long reads.
+We want to combine these approaches and use the SeqAn3 library as a basis for this new tool.
+
+## Current status:
+
+We can call insertions and deletions from long read data (SVIM methods implemented).
```
Comment (MitraDarja) [inline README.md:20]: Just as an idea, you could link your picture of your plan somewhere? (The one you showed last week), so a user knows what will go in next. :)

File: src/variant_detection/variant_detection.cpp [hunk 2]
```diff
@@ -167,8 +167,8 @@ void detect_junctions_in_long_reads_sam_file(std::vector<Junction> & junctions,
                         }
                     }
                     break;
-                case detection_methods::read_pairs: // There are no read pairs in long reads.
-                    break;
+                // There are no read pairs in long reads.
+                // case detection_methods::read_pairs
```
Comment (MitraDarja) [inline src/variant_detection/variant_detection.cpp:171]: Why is this change happening? :)

**Review by MitraDarja — APPROVED** [review]
Two things, but otherwise it LGTM. :D

**Comment by Irallia** [comment]
Bildschirmfoto 2021-07-05 um 14 01 22

**Review by joshuak94 — APPROVED** [review]
Looks good!

**Pull request merged**